code/espurna/scheduler.ino

```diff
@@ -136,7 +137,11 @@ void _schCheck() {
 
     time_t local_time = now();
     time_t utc_time = ntpLocal2UTC(local_time);
-
+    
+    if (getSetting("schAll").toInt() == 0) {
```

Comment:
`schEnabled`?

Reply:
schEnabled is already taken (its the toggle of each schedule), i changed to to schMasterEnabled so its more indicative